schwab/streaming.py

```diff
@@ -480,7 +481,7 @@ class ChartEquityFields(_BaseFieldEnum):
         VOLUME = 6
 
         #: Chart timestamp
-        CHART_TIME_MILLIS = 7
```

Comment:
Please do not rename fields that aren't labeled with placeholders. This risks breaking the lib for people whose apps were already written with the existing labels. Applies here and throughout this file.

Reply:
Understood.  Done.